stream-compositions/events/events/grandcentral/party-event.json

```diff
@@ -0,0 +1,284 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://syncservice.backbase.com/party.json",
+  "type": "object",
+  "description": "Event to create a new Party - v0.2.30",
+  "properties": {
+    "eventSource": {
+      "type": "string",
+      "description": "identifies the producer of this event."
+    },
+    "eventVersion": {
+      "type": "string",
+      "description": "The event (major) version. (eg v1, v2, v3)"
+    },
+    "identifier": {
+      "type": "string",
+      "description": ""
+    },
+    "dateTime": {
+      "type": "string",
+      "format": "date-time",
+      "description": "Timestamp of the event emitted from the core. e.g: 2023-11-13T20:20:39+00:00"
+    },
+    "dataContentType": {
+      "type": "string",
+      "default": "application/json"
+    },
+    "data": {
+      "type": "object",
+      "properties": {
+        "birthDate": {
+          "type": "string",
+          "format": "date"
+        },
+        "customFields": {
+          "type": "object",
+          "description": "Hosts custom fields related to party",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "dates": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "dateType": {
+                "type": "string",
+                "enum": [
```

Comment:
We took a different approach for `dates` like these, and in this case, this looks even weirder, because there is `birthDate` as well.

Reply:
This whole `data` spec was just a copy and paste from [party-api-v0.2.30.yaml](https://repo.backbase.com/artifactory/specs/unified-api/party/party-api-v0.2.30.yaml) specification, should we keep it consistent for now and only change it once the GC spec is updated?

Reply:
I would rather define an actual event... probably party-upsert ?

Reply:
party-upsert it will be!